vision/cloud-client/detect/set_endpoint.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+def set_endpoint():
+    """Change your endpoint"""
+    # [START vision_set_endpoint]
+    from google.cloud import vision
+
+    client_options = {'api_endpoint': 'eu-vision.googleapis.com:443'}
+
+    client = vision.ImageAnnotatorClient(client_options=client_options)
+    # [END vision_set_endpoint]
```

Comment:
In Go, and probably other languages, we have to include the entire sample. So I'm also including the part below but added some comments to try to make it explicit that the relevant part is to create the client only.

```go
// Create a client with a custom endpoint.
...

// Use the client with custom endpoint to detect texts on an image.
...
```

Do you think we should keep the code below in this sample as well for consistency? That would give users a runnable piece of code that does something.

Reply:
No, this is going on a special docs page and they don't want to show anything more than the above, I just have code below so that we can have a test.

Reply:
Got it, let me change it in the Go sample to return the client and do the rest in the test then.